app/src/main/java/com/github/sdpcoachme/profile/ProfileActivity.kt

```diff
@@ -138,12 +156,21 @@ class ProfileActivity : ComponentActivity() {
      */
     @Composable
     fun Profile(futureUserInfo: CompletableFuture<UserInfo>, isViewingCoach: Boolean) {
+
         val context = LocalContext.current
         var userInfo by remember { mutableStateOf(UserInfo()) }
+        var coachRating by remember { mutableStateOf(0)}
+        var currIsCoach by remember { mutableStateOf(false) }
```

Comment:
could be using the userInfo.coach variable instead of redefining another variable currIsCoach